src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1372,6 +1372,24 @@ public function getForeignKey()
         return Str::snake(class_basename($this)).'_'.$this->primaryKey;
     }
 
+    /**
+     * Returns a stable cache key that can be used to identify the record.
+     *
+     * @return string
+     */
+    public function cacheKey()
+    {
+        if (! $this->exists) {
+            return $this->getTable().'/new';
+        }
+
+        if (! $this->getAttribute('updated_at')) {
```

Comment:
I think this isn't correct: you can configure the actual name of the updated at column (think about existing schemas with different).

I believe this needs to call `getUpdatedAtColumn().